make.xsh

```diff
@@ -133,30 +133,55 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
     Use the "database-backup" service in the "docker-compose.yml" file to backup
     all of the tables in the instace of Postgres running locally.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
+    docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
```

Comment:
Would we use this makefile target on the lightsail instance to start the database for prod? If so, we'd want the auto-self-healing feature that restarts the database if it ever goes down. You can achieve that with `docker stack deploy --compose-file ... <stack_name>`. We'd probably call that stack `rootski-database`.

Reply:
Can you change this function to use docker swarm? You can use the `docker stack deploy ...` command from this comment ^^^

That may break the makefile target for manually doing a backup or a restore. You'll need to double check that those still work. 

This will also mean you'll need to double check that `make stop` still works. I think in that case, you'll need to add logic to `make stop` to do a `docker stack rm rootski-database` call. That way `make stop` can stop `make run` and `make start-local-db`.